tests/model_serving/model_server/kserve/negative/utils.py

```
@@ -65,7 +65,7 @@ def wait_for_isvc_model_status_states(
     *,
     target_model_state: str,
     transition_status: str,
-    timeout: int = 900,
+    timeout: int = Timeout.TIMEOUT_4MIN,
```

Flake8 (7.3.0)

[error] undefined name 'Timeout'

(F821)